moped-editor/src/components/GridTable/Filters.js

```diff
@@ -123,6 +132,57 @@ const generateEmptyField = (uuid) => {
   return { ...defaultNewFieldState, id: uuid };
 };
 
+/**
+ * Get configuration about the filters using the query parameters present in the URL
+ * @param {Array} filters - filters from the URL {field, operator, value}
+ * @param filtersConfig - The configuration object for the filters
+ * @return {Object}
+ */
+const makeInitialFilterParameters = (filters, filtersConfig) => {
+  if (filters.length === 0) return [];
+
+  return filters.reduce((acc, filter) => {
+    const filterUUID = uuidv4();
+    const { field, operator, value } = filter;
```

Comment:
These are the three values stored in the URL now. This function takes them and puts together all of the other config needed for the UI.